org.upbge.UPBGE.yml

```diff
@@ -0,0 +1,72 @@
+id: org.upbge.UPBGE
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: blender
+finish-args:
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--socket=pulseaudio"
+  - "--device=dri"
+  - "--share=network"
+  - "--filesystem=host"
+  - "--filesystem=/run/spnav.sock:ro"
+  - "--env=SPNAV_SOCKET=/run/spnav.sock"
+  - "--env=TMP_DIR=/tmp"
+  - "--env=TMP=/tmp"
+add-extensions:
+  org.upbge.UPBGE.Codecs:
+    directory: lib/codecs
+    add-ld-path: lib
+    bundle: true
+    autodelete: true
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '22.08'
+    add-ld-path: "."
+cleanup-commands:
+  - mkdir -p /app/lib/ffmpeg
+modules:
+  - shared-modules/libdecor/libdecor-0.1.1.json
+  - name: x264
+    config-opts:
+      - "--prefix=/app/lib/codecs"
+      - "--enable-lto"
+      - "--enable-pic"
+      - "--enable-shared"
+      - "--disable-cli"
+    sources:
+      - type: git
+        url: https://code.videolan.org/videolan/x264.git
+        commit: baee400fa9ced6f5481a728138fed6e867b0ff7f
+    cleanup:
+      - "/lib/codecs/include"
+      - "/lib/codecs/lib/pkgconfig"
+  - name: upbge
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 blender.sh /app/bin/blender
+      - mv blender /app/blender
+      - install -Dm644 /app/blender/org.upbge.UPBGE.desktop /app/share/applications/$FLATPAK_ID.desktop
```

Comment:
this is not found, hence the build failure

Reply:
Ah, makes sense- it's trying to build the latest tag. `org.upbge.UPBGE.desktop` only exists in the upstream since a few hours ago. Should I build the latest commit, or instead wait for a new tag?

Reply:
> Should I build the latest commit, or instead wait for a new tag?
I'm really wondering about this. Am I as a maintainer expected to prioritise metadata or stable releases of a package? If I want to get this package accepted earlier, should I temporarily keep the metadata in the Flatpak's repository? What about conflicting metadata across versions?

Reply:
> how would it be conficting if it comes from upstream?

It isn't in the latest tag of the upstream, so it isn't in any upstream release yet.

Reply:
then no conflict. and when it's shipped in the next release you can remove it here.